### PR TITLE
Disable APM and remove redundant logging init code

### PR DESCRIPTION
**Pull request by sprucely**

### CODE REVIEW

File: Keas.Mvc/Startup.cs [hunk 1]
```diff
@@ -218,11 +218,7 @@ public void ConfigureServices(IServiceCollection services)
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILoggerFactory loggerFactory, IHostApplicationLifetime appLifetime)
         {
-            app.UseAllElasticApm(Configuration);
-            
-            // setup logging
-            app.ConfigureStackifyLogging(Configuration);
```
Comment (jSylvestre) [inline Keas.Mvc/Startup.cs:224]: Did you mean to delete these?

Comment (srkirkland) [reply Keas.Mvc/Startup.cs:224]: Agreed I would think you'd still need the serilog entry at least.

File: Keas.Mvc/Program.cs [hunk 2]
```diff
@@ -89,7 +89,7 @@ static LoggerConfiguration CreateLogConfig(IConfigurationRoot configuration)
                 .MinimumLevel.Override("Microsoft", LogEventLevel.Information)
                 // .MinimumLevel.Override("Microsoft.EntityFrameworkCore", LogEventLevel.Warning) // uncomment this to hide EF core general info logs
                 .MinimumLevel.Override("Microsoft.AspNetCore", LogEventLevel.Warning)
-                .Enrich.WithElasticApmCorrelationInfo()
+                //.Enrich.WithElasticApmCorrelationInfo()
```
Comment (srkirkland) [inline Keas.Mvc/Program.cs:92]: Does it add much to the logs if we leave the correlation? I think if we remove our stackify logger we would want to keep this apm enriched.

**Review by srkirkland — CHANGES_REQUESTED** [review]

File: Keas.Mvc/Startup.cs (hunk 1 above)
Comment (sprucely) [reply Keas.Mvc/Startup.cs:224]: `ConfigureStackifyLogging()` is called in `Program.cs`
`loggerFactory.AddSerilog()` is the old way of doing it, and is handled internally by `IHostBuilder.UseSerilog()` in `Program.cs`

**Pull request merged**